fractions/keycloak/src/main/java/org/wildfly/swarm/keycloak/internal/SecuredImpl.java

```diff
@@ -65,28 +67,12 @@ public SecuredImpl(ArchiveBase<?> archive) {
         getArchive().as(JARArchive.class).addModule("org.wildfly.swarm.keycloak", "deployment");
         getArchive().as(JARArchive.class).addAsServiceProvider("io.undertow.servlet.ServletExtension", "org.wildfly.swarm.keycloak.deployment.SecurityContextServletExtension");
 
-        InputStream keycloakJson = Thread.currentThread().getContextClassLoader().getResourceAsStream("keycloak.json");
-        if (keycloakJson == null) {
-
-            String appArtifact = System.getProperty(BootstrapProperties.APP_ARTIFACT);
-
-            if (appArtifact != null) {
-                try (InputStream in = ClassLoader.getSystemClassLoader().getResourceAsStream("_bootstrap/" + appArtifact)) {
-                    Archive tmpArchive = ShrinkWrap.create(JARArchive.class);
-                    tmpArchive.as(ZipImporter.class).importFrom(in);
-                    Node jsonNode = tmpArchive.get("keycloak.json");
-                    if (jsonNode == null) {
-                        jsonNode = tmpArchive.get("WEB-INF/keycloak.json");
-                    }
-
-                    if (jsonNode != null && jsonNode.getAsset() != null) {
-                        keycloakJson = jsonNode.getAsset().openStream();
-                    }
-                } catch (IOException e) {
-                    // ignore
-                    // e.printStackTrace();
-                }
-            }
+        InputStream keycloakJson;
+        String keycloakJsonPath = System.getProperty("swarm.keycloak.json.path");
```

Comment:
It would be nice to use @Configurable here. The information might come from `project-stages.yml` too